models/balancer/gnosis/balancer_v2_gnosis_bpt_prices.sql

```diff
@@ -19,7 +20,7 @@ WITH
         WHERE CAST(tokenIn AS VARCHAR(66)) = SUBSTRING(CAST(poolId AS VARCHAR(66)), 1, 42)
```

Comment:
Change to: 
`WHERE tokenIn = bytearray_substring(poolId, 1, 20)
 OR tokenOut = bytearray_substring(poolId, 1, 20)`

Reply:
replace line 33 with:
`CAST(bytearray_substring(a.poolId, 1, 20) AS VARCHAR) AS bpt_address,`

Reply:
thanks for the review, @thetroyharris!